src/components/riddleDetail/InfoAccordion.tsx

```diff
@@ -37,7 +37,7 @@ export const InfoAccordion: FC<Props> = ({ label, solvers }) => {
 
 	return (
 		<>
-			<Accordion>
+			<Accordion sx={{ maxWidth: { xs: '300px', sm: 'none' } }}>
```

Comment:
Má to tu nějaký význam?

Reply:
Jo, kvůli hodně malým telefonům, jinak to přetékalo.

Reply:
Jaká šířka? Zobrazuje se mi to v pohodě až do úplně malých šířek.

Reply:
375x812  - IPhone 11 14.6